package.json

```
@@ -5,6 +5,7 @@
   "version": "2.22.0",
   "scripts": {
     "preinstall": "node ./bin/check-nodejs-version.js",
+    "prepare": "husky",
```

I'm not really understanding the point of the Husky projects when we already have `git config --local core.hooksPath .githooks`. I tested it and it seems to work alright.
Maybe we can avoid it?

```suggestion
    "prepare": "git config --local core.hooksPath .githooks",
```

Also maybe this should be optional, and specified in DEVELOPMENT.md.